### DISCUSSION TITLE
How to show name of additional entity states?

### DISCUSSION THREAD

**Discussion by mrtnnwmn** [kalkih/mini-graph-card#1017] [Q&A]
I'm using a mini graph card to show the actual & target temperature of a radiator. I can show the state of the second entity in the card by setting `show_state: true`, but is it possible to also show the name of the entity above the state?

This is how it currently shows:
![image](https://github.com/kalkih/mini-graph-card/assets/89987410/1e02389f-7a05-4e2e-8b7a-4dc5e921bec2)

This is what I'd like it to show:
![image](https://github.com/kalkih/mini-graph-card/assets/89987410/47f8a6ff-e518-4b4c-9a56-8746570d551e)

**Comment by ildar170975** [comment]
card-mod is your only way.
Go to card-mod Community thread for examples

**Reply by mrtnnwmn** [reply]
I've had a look but I can't find any examples that are quite the same, and playing around with `.states--secondary::before` has got me this:
![image](https://github.com/kalkih/mini-graph-card/assets/89987410/cb418f19-90c3-4ed2-89d1-c1e612442015)

Unfortunately trying a `before` on the first state results in this:
![image](https://github.com/kalkih/mini-graph-card/assets/89987410/9f2275a5-cab0-47e4-bf51-c762a9143ab9)
